commcare_connect/opportunity/forms.py

```diff
@@ -408,170 +458,6 @@ def clean(self):
             return cleaned_data
 
 
-class OpportunityCreationForm(forms.ModelForm):
```

Comment:
This flow has not been maintained since we got the new Opportunity Creation flow. I have removed it in this PR, also removed any tests that were related to this.

Reply:
nice